README.md

```diff
@@ -1,130 +1,191 @@
 
-You may embed `callback.js` in the page that is served by your redirectUri. Upon loading, this file will invoke the `onComplete` callback specified above and close the window:
 ```html
 <script src="https://cdn.smartcar.com/javascript-sdk/callback-1.0.0.js"></script>
 ```
 
-To retrieve the Smartcar OAuth URL (useful in single page applications when redirecting to Smartcar's OAuth flow in a new tab):
-```javascript
-// note: state and forcePrompt are optional
-smartcar.generateLink({state, forcePrompt});
-```
-
 ## Installation
+
 You can import this SDK into your application from Smartcar's CDN:
 
 ### sdk.js
+
 ```html
 <script src="https://cdn.smartcar.com/javascript-sdk/sdk-1.0.0.js"></script>
 ```
 
-### callback.js
+### redirect.js
+
 ```html
 <script src="https://cdn.smartcar.com/javascript-sdk/callback-1.0.0.js"></script>
 ```
 
 ## Configuration
+
 ### `new Smartcar({options})`
+
 #### Options:
 | Parameter       | Type | Description   |
 |:--------------- |:---|:------------- |
-| `clientId`      | String |Application clientId obtained from [Smartcar Developer Portal](https://developer.smartcar.com). If you do not have access to the dashboard, please [request access](https://smartcar.com/subscribe). |
-| `redirectUri`   | String |**Required** RedirectURI set in [application settings](https://developer.smartcar.com/apps). Given URL must match URL in application settings. |
+| `clientId`      | String |Application clientId obtained from [Smartcar Developer Portal](https://dashboard.smartcar.com). |
+| `redirectUri`   | String |**Required** Redirect URI set in [application settings](https://dashboard.smartcar.com/apps). Given URL must match URL in application settings. To use Smartcar hosting this URI must be of the form `https://javascript-sdk.smartcar.com/redirect?app_origin=${your client URI here}` |
 | `scope`         | String[] |**Optional** List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs#get-all-vehicles). |
-| `onComplete`      | Function |**Optional** Function to be invoked on completion of the Smartcar authorization flow. This function will only be invoked if `callback.js` is loaded in the page served by your redirectUri. |
+| `onComplete`      | Function |**Optional** Function to be invoked on completion of the Smartcar authorization flow. This function will only be invoked if `redirect.js` is loaded in the page served at your redirect URI. |
 | `development`   | Boolean |**Optional** Launch Smartcar auth in development mode to enable the mock vehicle brand. |
 
+
+<!-- TODO: I think it might be useful to consolidate the three instance methods reference as they are so similar outside of the `id` parameter to `addClickHandler` but want to get others' thoughts before doing so -->
```

Comment:
seems annoying to have to update in multiple locations, but do feel its more dev friendly to see it on each method they are looking for rather than another place. ideally, we should probably just generate this all out using JSDoc

Reply:
I was thinking less so from a maintenance perspective and more that as a prospective developer I think I'd prefer not to have such repetitive docs but your comment seems to answer my question about whether that's a universal thought 😄